_docs/3x/execution/scheduler.md

```diff
@@ -319,13 +323,11 @@ Currently there are 3 execution models available:
   indefinitely, especially on top of Javascript where a long loop
   can mean that the UI gets frozen and where we need to be cooperative.
 - [AlwaysAsyncExecution]({{ page.path | api_base_url }}monix/execution/ExecutionModel$.html#AlwaysAsyncExecution)
-  specifies that units of work within a loop should always execute
-  asynchronously on each step, being basically the mode of operation
-  for Scala's `Future`.
+  specifies that each task should be treated as if it were asynchronous,
+  being basically the mode of operation for Scala's `Future`.
 - [SynchronousExecution]({{ page.path | api_base_url }}monix/execution/ExecutionModel$.html#SynchronousExecution)
-  specifies that synchronous execution should always be preferred,
-  for as long as possible, being basically the mode of operation
-  for the Scalaz `Task`.
+  specifies that tasks should be executed synchronously for as long
+  as possible, being basically the mode of operation for the Scalaz `Task`.
```

Comment:
How about we change `Scalaz Task` to `cats.effect.IO` ? It should be more recognizable nowadays and Scalaz Task could be confused with ZIO which uses the same model as Monix Task by default

Reply:
Done 👍